tools/setup_pf.py

```diff
@@ -58,8 +58,8 @@ def setup_configs():
 
     # NOTE: the following two CAPTCHA keys are dummy keys for testing only.
     # They should be replaced with real keys upon launch.
-    config.set(captcha_private_key='6LfiOr8SAAAAAFyxGzWkhjo_GRXxYoDEbNkt60F2',
-               captcha_public_key='6LfiOr8SAAAAAM3wRtnLdgiVfud8uxCqVVJWCs-z',
+    config.set(captcha_site_key='6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI',
```

Comment:
Are these keys specific to your host name, or can be used by anyone? If other people cannot use it, it would be better to delete them.